jacobian_fixes.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/flowtracks/interpolation.py b/flowtracks/interpolation.py
```

Comment:
Why do you need to add the patch to the repository?

Reply:
To be sure I did the right thing :) 

Will remove it